src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ExportClient/BaseOtlpHttpExportClient.cs

```diff
@@ -46,17 +46,17 @@ protected BaseOtlpHttpExportClient(OtlpExporterOptions options, HttpClient httpC
         /// <inheritdoc/>
         public bool SendExportRequest(TRequest request, CancellationToken cancellationToken = default)
         {
+            using var httpRequest = this.CreateHttpRequest(request);
+
             try
             {
-                using var httpRequest = this.CreateHttpRequest(request);
-
                 using var httpResponse = this.SendHttpRequest(httpRequest, cancellationToken);
 
                 httpResponse?.EnsureSuccessStatusCode();
             }
             catch (HttpRequestException ex)
             {
-                OpenTelemetryProtocolExporterEventSource.Log.FailedToReachCollector(this.Endpoint, ex);
+                OpenTelemetryProtocolExporterEventSource.Log.FailedToReachCollector(httpRequest.RequestUri, ex);
```

Comment:
This change is to ensure that the log has the actual path that it attempted to use.